### ISSUE TITLE
Error with Kraken2 database

### ISSUE THREAD

**Issue by ehj000** [usegalaxy-no/galaxyadmin#45]
The Kraken2 tools gives the following error: 
kraken2: database ("/cvmfs/data.galaxyproject.org/managed/kraken2_databases/2020-11-26T021706Z_standard_kmer-len_35_minimizer-len_31_minimizer-spaces_6_load-factor_0.7") does not contain necessary file taxo.k2d

I have checked the server and taxo.k2d is on this location. I suspect that the database was not built properly upon installation of the tool, but not sure how to re-build the database on the galaxy server (kraken2-build --standard on a "normal" installation)

**Comment by kjellp** [comment]
Does the file access attributes look ok?

Are you thinking on building a new versjon manually yourself to 
"our own "  cvmfs , and present it to the tool in its own loc file ?  

The dataases On the global cvmfs Were not built  as part  of any installation process on Our servers . They were preprocessed and provided by others. Maybe a quick kraken2 - build job on your l ocal Computer / Server can give a clue to if your hypothesis is correct by comparring the outputs ?

**Comment by ehj000** [comment]
Might be some issues with the cvmfs. I tested mapping of reads using BWA against the e. coli reference:

[E::bwa_idx_load_from_disk] fail to locate the index files

Location of index: /cvmfs/data.galaxyproject.org/managed/bwa_mem_index/eschColi_K12/eschColi_K12.fa

**Comment by kjetilkl** [comment]
Yes, it seems the "/cvmfs/data.galaxyproject.org/" directory is not available on the slurm node.

**Comment by ehj000** [comment]
Hmm. Can we make it available or do we need to host the indexes locally? This will only resolve the problem for this particular tool

**Comment by kjetilkl** [comment]
This is a problem that affects all tools that use data from "/cvmfs/data.galaxyproject.org/", so we have to make it available again. It worked before, so I don't know why it suddenly has gone missing now.
